Sqlite/ImportSql.py

```diff
@@ -251,7 +251,7 @@ def get_media_list(self, sql, from_type, from_handle):
         return [self.pack_media_ref(sql, result) for result in results]
 
     def get_surname_list(self, sql, handle):
-        results = sql.query("""select * from surname where handle = ?;""", handle)
+        results = sql.query("""select s.* from surname s inner join link l ON l.to_handle = s.handle where l.from_handle = ?;""", handle)
```

Comment:
I think you can use `.get_links()` instead of the join. `links` should probably have an explicit ordering. But that can be a separate fix. Thanks!